_gsdocs-orgs/2019/cern.md

```diff
@@ -0,0 +1,13 @@
+---
+title: "CERN"
+author: "Benedikt Hegner"
+layout: default
+organization: CERN
+logo: CERN-logo.jpg
+description: |
+  At CERN, the European Organization for Nuclear Research, physicists and engineers are probing the fundamental
+  structure of the universe. They use the world's largest and most complex scientific instruments to study the
```

Comment:
There is a missing space between fundamental and structure

Reply:
Another missing space between 'study the' and 'basic constituents'

Reply:
@aflp91 I checked twice here and in my local changes, but I don't see any missing space. 
I will mark this as resolved, please feel free to re-open if needed.

Reply:
I've pulled again and I still see them… Look at:
    https://owncloud.lal.in2p3.fr/index.php/s/u0R5JS8h8HE126x

Reply:
@JavierCVilla if you run your branch in Jekyll, you'll see the problem mentioned by @aflp91, despite the source file seems correct (as far as I can see). I'd suggest that you add a space at the end of each line of the CERN description (except the last one) so that when they are merged it is properly added (tested).

Reply:
@aflp91 @jouvin I did not realize of this issue, nice catch. 

Indeed it was wrong in several places where this description appeared, now should be fixed.